frontend/src/components/TripInfoComponents/TripInfoContainer.js

```diff
@@ -46,70 +41,14 @@ class TripInfoContainer extends React.Component {
     axiosConfig
       .get(`/trip/${this.props.match.params.tripId}`)
       .then(res => {
+        console.log(res);
         this.setState({
           ...res.data,
           currentDestination: res.data.destinations[0]
         });
       })
       .catch(err => console.log(err));
   };
-  //check / uncheck list items
-  togglePacking = item => {
-    const newItem = { item: item.item, done: !item.done };
-    // const newPacking = [...this.state.packing];
-    // newPacking.push({ item: item.item, done: !item.done });
-    // newPacking.splice(index, 1);
-    // this.setState({ packing: newPacking });
-    // this.updateItem(this.state.packing, "packing");
-    console.log("toggle packing!");
-    this.updateItem(newItem, "packing");
-  };
-
-  toggleTodo = item => {
-    const newItem = { item: item.item, done: !item.done };
-    this.updateItem(newItem, "todos");
-  };
-
-  //add list items
-  addPacking = ({ item }) => {
-    // const newList = [...this.state.packing];
-    // newList.push({ item: item.item, done: false });
-    // this.setState({ packing: newList });
-    const newItem = { item, done: false };
-    console.log("newItem: ", newItem);
```

Comment:
Leave the deleted code in this file for now please

Reply:
I'm not sure where all those changes came from. The only files I changed were Login.js and Welcome.js. How do I get the rest to revert back? I did a push to my branch and a pull from master. The same two things I've been doing since the start of the project. @kyullog